src/pages/admin/problems/add.page.tsx

```diff
@@ -0,0 +1,136 @@
+import React from "react"
+import "@uiw/react-md-editor/markdown-editor.css"
+import "@uiw/react-markdown-preview/markdown.css"
+import "katex/dist/katex.css"
+import dynamic from "next/dynamic"
+import { useState } from "react"
+import katex from "katex"
+import {
+   Box,
+   DescriptionTextField,
+   TitleTextField,
+   HorizaontalBox,
+   Card,
+   TextField,
+} from "./styles"
+import { useServerManager } from "../../../api/server"
+import { ProblemDTO } from "../../../types/ProblemDTO"
+import { Button, CardContent, CardHeader, Divider } from "@mui/material"
+
+const MDEditor = dynamic(() => import("@uiw/react-md-editor"), { ssr: false })
+
+export default function AddProblem(): JSX.Element {
+   const mdKaTeX = `This is to display the 
+\`$$c = \\pm\\sqrt{a^2 + b^2}$$\`
+ in one line
+\`\`\`KaTeX
+c = \\pm\\sqrt{a^2 + b^2}
+\`\`\`
+\`\`\`KaTeX
+\ \\relax{x} = \\int_{-\\infty}^\\infty
+    \ \\hat\\xi\\,e^{2 \\pi i \\xi x}
+    \\,d\\xi
+\`\`\`
+`
+   const serverManager = useServerManager()
+   const [problemData, setProblemData] = useState<ProblemDTO>({
+      dataSets: [],
+      limits: {
+         memory: 0,
+         time: 0,
+      },
+      text: mdKaTeX,
+      title: "",
+      description: "",
+   })
+
+   const handleUpload = () => {
+      const data = new FormData()
+
+      Object.entries(problemData).forEach((entie) => {
+         data.append(entie[0], entie[1])
+      })
+
+      serverManager.createProblem(data)
+   }
+
+   const disableButton =
```

Comment:
The forma should be validate using some solution to do that. For example formstate